bin/snapshot-backups/db-dump-push-to-snapshot

```diff
@@ -1,28 +1,24 @@
 #!/usr/bin/env bash
 
-# we might be running in circleci
-if [ -f /home/circleci/project/env.local ]; then
-  . /home/circleci/project/env.local
-fi
-# we might be running from a local dev machine
-SCRIPT_DIR="$(dirname "$0")"
-if [ -f $SCRIPT_DIR/env.local ]; then
-  . $SCRIPT_DIR/env.local
-fi
-if [ -f ./env.local ]; then
-  . ./env.local
-fi
-if [ -f $SCRIPT_DIR/../deploy/includes ]; then
-  . $SCRIPT_DIR/../deploy/includes
+usage="
+$0: Pushes a local a backup of a database to current environment.
```

Comment:
Let's say "current environment's backup storage." "to current environment" sounds like it might overwrite the running database.